instrumentation/trace/context.go

```diff
@@ -24,6 +25,34 @@ type Context struct {
 	requestId string
 }
 
+const RequestTraceName = "X-ORBS-NAME"
+const RequestTraceTime = "X-ORBS-CREATED"
+const RequestTraceRequestId = "X-ORBS-ID"
+
+func NewFromRequest(ctx context.Context, request *http.Request) context.Context {
+	name := request.Header.Get(RequestTraceName)
+	if name != "" {
```

Comment:
As a general rule, better check `if len(name)==0` and then `return ctx` immediately, rather than enclosing the whole function body within the `if` statement - this reduces indentation thus making the code more readable.

Reply:
I agree and disagree. No need for `len(name)` - but check `if name == ""`